astrocut/utils/wcs_fitting.py

```diff
@@ -20,197 +20,249 @@
 from astropy import units as u
 from astropy.wcs.utils import celestial_frame_to_wcs
 
-def _linear_transformation_fit(params, lon, lat, x, y, w_obj):  # pragma: no cover
-    """ Objective function for fitting linear terms."""
-    pc = params[0:4]
+def _linear_wcs_fit(params, lon, lat, x, y, w_obj):
+    """
+    Objective function for fitting linear terms.
+
+    Parameters
+    ----------
+    params : array
+        6 element array. First 4 elements are PC matrix, last 2 are CRPIX.
+    lon, lat: array
+        Sky coordinates.
+    x, y: array
+        Pixel coordinates
+    w_obj: `~astropy.wcs.WCS`
+        WCS object
+        """
+    cd = params[0:4]
```

Comment:
A little dangerous to access parameters directly on indexes, if params[] changes structure in the future this would need to be updated and could be forgoten about.  Minor comment though.